aced_submission/grip_load.py

```diff
@@ -222,6 +222,26 @@ def delete_project(graph_name: str, project_id: str, output: dict, access_token:
     return json_data
 
 
+def get_project_data(graph_name: str, project_id: str, output:dict, access_token: str)-> Generator[dict, None, None]:
+    """Retrieves all of the data for a given project id on a given graph"""
+    response = requests.get(
+        f"http://{GRIP_SERVICE}/{NGINX_PATH}/{graph_name}/get-vertices/{project_id}",
+        headers={"Authorization":f"bearer {access_token}"}
+    )
+    def stream_res(response):
+        for result in response.iter_lines(chunk_size=None):
+            try:
+                result_dict = orjson.loads(result.decode())
+            except Exception as e:
+                print("Failed to decode: %s", result)
+                raise e
+            yield result_dict["data"]
+
+    return stream_res(response)
+
+
+"""WARNING The functions below use the GRIP protobuf API directly and bipass Auth checks
+            and should not be used for Gen3 Purposes."""
```

Comment:
do we wanna delete this then or do we have an internal use case for it?

Reply:
I don't know. But I also don't want to rewrite this when the use case pops up.